lib/new_playground.dart

```diff
@@ -236,10 +236,15 @@ class Playground implements GistContainer, GistController {
 
     // Helper function to create LIElement with correct attributes and classes
     // for material-components-web
-    LIElement _menuElement(String gistId, String name) {
+    LIElement _menuElement(String gistId, String name, Layout layout) {
```

Comment:
nit: Adding a parameter makes sense, but now that there's a Sample class, this function should probably just take a single parameter of type `Sample`.

For the record, you're always free to refactor things like that when making changes. Don't feel like you can only incrementally add to what we've already got! 😄